test/ui/FieldTableTest.ts

```diff
@@ -64,7 +64,7 @@ export function FieldTableTest() {
         });
 
         it('should show a toggle link', function () {
-          expect($$(test.env.element.parentElement).find('.coveo-field-table-toggle')).not.toBeNull();
+          expect($$(test.env.element.parentElement.parentElement).find('.coveo-field-table-toggle')).not.toBeNull();
```

Comment:
It would have been nicer to determine why we need to go parentElement.parentElement for sure, instead of just fixing it quickly like that just so the test pass :)

Are we sure that it's okay ? Is the test flawed ? Why using `test.env.root` is not working ? etc.